addons/binding/org.openhab.binding.lutron/ESH-INF/thing/keypads.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Well, the other option would be to have only the bridges/channels in thing-types.xml and have a separate file for all "product", e.g. also for the dimmers, etc.
But I do not really have any preferences, so feel free to split it how you feel it is best. It just seemed unnatural to have some devices within the one file while others are in a separate file.